docs/contributing.md

```diff
@@ -0,0 +1,108 @@
+# Contributing to Kusion
+
+Welcome to Kusion! Kusion is a cloud-native application delivery and operation engine, which is designed to simplify and optimize the process of delivering applications for dev and platform. Kusion belongs to [KusionStack](https://github.com/KusionStack), which is a technology stack in the field of platform engineering.
+
+We warmly appreciate your talent and creativity in contributing to this project. This guide will help direct you to the best places to start contributing. Follow the instructions below, you'll be able to pick up issues, write code to fix them, and get your work reviewed and merged.
+
+Feel free to create issues and contribute your code. Whether you are an experienced developer or just beginning your journey in the open-source world, we highly encourage your participation.
+
+If you have any questions or need further information, please don't hesitate to contact us.
+
+---
+
+<!-- MarkdownTOC autolink="true" -->
+
+- [File an Issue](#file-an-issue)
+- [Contribute a Pull Request](#contribute-a-pull-request)
+    - [Fork Repository](#fork-repository)
+    - [Develop Code](#develop-code)
+    - [Open a Pull Request](#open-a-pull-request)
+    - [Sign CLA](#sign-cla)
+    - [PR Checks](#pr-checks)
+
+## File an Issue
```

Comment:
New contributors can also pick an existing issue instead of filing a new one. Please add a section about choosing an existing issue, such as issues with `good first issue` or `help wanted` labels.

Reply:
Can you provide one?

Reply:
Provide what? issues? You can select issues with these labels
![image](https://github.com/KusionStack/kusion/assets/4793557/bb6773db-3c55-416a-a9fb-6d995f93fdc9)